Add redisinsight v1.13.1 release notes. (#2225)
RedisLabs/redislabs-docs: content/ri/release-notes/v1.13.0.md

weight: 4
---

## 1.13.1 (November 2022)

This is the maintenance release of RedisInsight 1.13 (v1.13.1)!

### Fixes:
- Core:
  - Fixed container vulnerabilities.

>   - Disable healthcheck API to overload RedisInsight DB.

Reviewer: Mind explaining this fix a bit further? What was happening?

Author: done

>   - Disable healthcheck API to overload RedisInsight DB. Earlier a separate session was created for each healthcheck hit which overloaded the database with too many session tokens. Now healtcheck API doesn't create any session token.

Reviewer: ```suggestion
  - Prevented healthcheck API from overloading RedisInsight DB. Earlier, a separate session was created for each healthcheck hit, which overloaded the database with too many session tokens. Now, healtcheck API doesn't create any session tokens.
```
